lib/vagrant-service-manager/installer.rb

```diff
@@ -0,0 +1,28 @@
+require_relative 'adb_executor'
+require_relative 'cdk_executor'
+
+module VagrantPlugins
+  module ServiceManager
+    module Executor
+      def self.build(type, machine, ui, options)
```

Comment:
+1, I suggest you align with what I am doing in vagrant-registration. This was we are consistent between plugins and it is easier to understand/extend.